src/main/java/org/gridsuite/modification/server/entities/equipment/modification/ShuntCompensatorModificationEntity.java

```diff
@@ -0,0 +1,92 @@
+import javax.persistence.AttributeOverrides;
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+import static org.gridsuite.modification.server.dto.AttributeModification.toAttributeModification;
+
+/**
+ * @author Seddik Yengui <Seddik.yengui at rte-france.com>
+ */
+
+@NoArgsConstructor
+@Getter
+@Entity
+@Table(name = "shuntCompensatorModification")
+public class ShuntCompensatorModificationEntity extends InjectionModificationEntity {
+    @Embedded
+    @AttributeOverrides(value = {
+            @AttributeOverride(name = "value", column = @Column(name = "susceptancePerSection")),
+            @AttributeOverride(name = "opType", column = @Column(name = "susceptancePerSectionOp"))
+    })
+    private DoubleModificationEmbedded susceptancePerSection;
+
+    @Embedded
+    @AttributeOverrides(value = {
+            @AttributeOverride(name = "value", column = @Column(name = "qAtNominalV")),
+            @AttributeOverride(name = "opType", column = @Column(name = "qAtNominalVOp"))
+    })
+    private DoubleModificationEmbedded qAtNominalV;
+
+    @Embedded
+    @AttributeOverrides(value = {
+            @AttributeOverride(name = "value", column = @Column(name = "shuntCompensatorType")),
+            @AttributeOverride(name = "opType", column = @Column(name = "shuntCompensatorTypeOp"))
+    })
+    private EnumModificationEmbedded<ShuntCompensatorType> shuntCompensatorType;
+
+    public ShuntCompensatorModificationEntity(ShuntCompensatorModificationInfos shuntCompensatorModificationInfos) {
+        super(shuntCompensatorModificationInfos);
+        assignAttributes(shuntCompensatorModificationInfos);
+    }
+
+    @Override
+    public void update(@NonNull ModificationInfos modificationInfos) {
+        super.update(modificationInfos);
+        assignAttributes((ShuntCompensatorModificationInfos) modificationInfos);
+    }
+
+    private void assignAttributes(ShuntCompensatorModificationInfos shuntCompensatorModificationInfos) {
+        this.qAtNominalV = new DoubleModificationEmbedded(shuntCompensatorModificationInfos.getQAtNominalV());
+        this.shuntCompensatorType = new EnumModificationEmbedded<>(shuntCompensatorModificationInfos.getShuntCompensatorType());
+        this.susceptancePerSection = new DoubleModificationEmbedded(shuntCompensatorModificationInfos.getSusceptancePerSection());
+    }
+
+    @Override
+    public ShuntCompensatorModificationInfos toModificationInfos() {
+        return toShuntCompensatorModificationInfosBuilder().build();
+    }
+
+    private ShuntCompensatorModificationInfos.ShuntCompensatorModificationInfosBuilder<?, ?> toShuntCompensatorModificationInfosBuilder() {
+        return ShuntCompensatorModificationInfos
+                .builder()
+                .uuid(getId())
+                .date(getDate())
+                .voltageLevelId(toAttributeModification(getVoltageLevelIdValue(), getVoltageLevelIdOp()))
```

Comment:
we need voltageLevelId, but it's strange to get it with a modifiable field (value/op couple) because we can not change it with the form (it is not displayed either).